Chapitres/Ch12/Ch12_source.qmd

```diff
@@ -45,7 +45,7 @@ getInput <- function(){
 data.dt <- data.table(
   x1=getInput(),
   x2=getInput())
-library(animint2)
+library(animint2fr)
```

Comment:
animint2fr is a data package, with only BanqueMondiale https://github.com/animint/animint2fr/tree/main/data

so I think library(animint2) should be kept here.